misc/iitthesis.cls

```diff
@@ -607,6 +607,7 @@
   \renewcommand{\glossarysection}[2][]{}
   \newcommand{\makenotationAndAbbreviations}{%
    \chapter*{Notation and Abbreviations}
+   \label{chap:abbrev}
```

Comment:
Why not `\label{chap:notations-and-abbrevations}` ?

Reply:
Sure, I'll change that.